demos/dermatology.html

```diff
@@ -309,6 +311,7 @@
                             displaySerialization(currWidget);
                             displaySerializationText(currWidget);
                             displayThemeText(currWidget);
+                            frame.doResize();

```

Comment:
At one point this was needed otherwise the labels were cut off until after the first update. This issue doesn't seem to be happening anymore. I'll remove this.